local/bin/sh/run-site-no-pre-build.sh

```diff
@@ -1,20 +1,21 @@
 #!/bin/bash
 
 RUN_SERVER=${RUN_SERVER:=false}
-RUN_WEBPACK=${RUN_WEBPACK:=true}
 RENDER_SITE_TO_DISK=${RENDER_SITE_TO_DISK:=false}
 
 if [ ${RUN_SERVER} = true ]; then
+  yarn run start
+  
   # Building the documentation
-  if [ ${RUN_WEBPACK} = true ]; then
-    printf "checking that node modules are installed and up-to-date"
-    npm --global install yarn && \
-    npm cache clean --force && yarn install --frozen-lockfile
-    printf "starting webpack and hugo build"
-	  yarn run start
+  # if [ ${RUN_WEBPACK} = true ]; then
+  #   printf "checking that node modules are installed and up-to-date"
+  #   npm --global install yarn && \
+  #   npm cache clean --force && yarn install --frozen-lockfile
+  #   printf "starting webpack and hugo build"
+	#   yarn run start
 
-    sleep 5
-	fi
+  #   sleep 5
+	# fi
```

Comment:
I think we may still need the `yarn install --frozen-lockfile`, you can test by deleting your node modules dir and running make start

Reply:
@davidejones thanks!  i have commented back in the previous script that was used here and `run-site.sh`.   it works when deleting `node_modules` and re-starting server.